schultz_a03b.js

```diff
@@ -0,0 +1,77 @@
+/*Chris Schultz
+schultz_a03b.js
+Info 2124 WW
+Thoendel
+12-16-2019*/
+
+//delcaring global constants
+const firstName = 'Chris';  //3a
+const lastName = 'Schultz';  //3b
+const villainFirstName = 'Cruella'; //3c
+const villainLastName = 'Deville'; //3d
+const companyName = 'Mutual of Omaha' //3e
+
+//beginning block as requirements ask for block-scoped variables
+{
```

Comment:
you didn't necessarily have to have a block here - just the variables had to support block scoping. In other words - use let - not var.